client/src/app/__tests__/AppSpec.js

```diff
@@ -735,6 +735,27 @@ describe('<App>', function() {
       expect(closeTabResponse).to.eql(false);
     });
 
+
+    it('should emit <app.tabsChanged> event on tab closed', async function() {
```

Comment:
Should this also emit when we change a tab? (ie. click on an already opened tab?) In this case, we miss a test case I believe?

Currently we only test:
* tab close
* tab create

Reply:
👍  I added a test case via https://github.com/camunda/camunda-modeler/pull/2631/commits/189ef000355015fa9a7d75440639a79e7992535c accordingly.